docs/bigcommerce-for-wordpress/extending-the-plugin/customization-guide.md

```diff
@@ -289,20 +385,21 @@ function myCheckoutFunction($checkout_config) {
   $checkout_config['styles']['step']['color'] = '#0BE640'; //green
   return $checkout_config;
 }
-add_filter( 'bigcommerce/checkout/config', 'myCheckoutFunction');
-
+add_filter('bigcommerce/checkout/config', 'myCheckoutFunction');
 ```
 
 Following this format, you can apply styles to other elements, like buttons, input fields, and checkboxes. See the full list of checkout elements that can be styled and which properties you can adjust in the [Embedded Checkout Styles documentation](https://github.com/bigcommerce/checkout-sdk-js/blob/master/docs/interfaces/embeddedcheckoutstyles.md) (Github).

```

Comment:
Following this format, you can apply styles to other elements, like buttons, input fields, and checkboxes. See the full list of checkout elements that you can style and which properties you can adjust in the [Embedded Checkout Styles documentation](https://github.com/bigcommerce/checkout-sdk-js/blob/master/docs/interfaces/embeddedcheckoutstyles.md) (Github).